qiita_db/metadata_template.py

```diff
@@ -1965,7 +1993,29 @@ def load_template_to_dataframe(fn):
                            keep_default_na=False, na_values=[''],
                            parse_dates=True, index_col=False, comment='\t',
                            mangle_dupe_cols=False, converters={
-                               'sample_name': lambda x: str(x).strip()})
```

Comment:
Instead of hardcoding these types, can we get them from the database? See [here](http://stackoverflow.com/questions/2146705/select-datatype-of-the-field-in-postgres). This way if something on the db changes, we will not need to update this code.